### PR TITLE
[BEAM-8377] Remove the flink-1.6 from the project-mappings

### PR DESCRIPTION
**Pull request by sunjincheng121**
Flink 1.5 and 1.6 have been drop in #9632, So we should remove the flink-1.6 from the project-mappings.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python2/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python36/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python37/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python2_PVR_Flink_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python35_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)
XLang | --- | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_XVR_Flink/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PreCommit_PythonLint_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by sunjincheng121** [comment]
R: @adude3141

File: project-mappings [hunk 1]
```diff
@@ -100,9 +100,6 @@
 :beam-runners-flink-1.7-job-server-container :runners:flink:1.7:job-server-container
 :beam-runners-flink-1.7-job-server :runners:flink:1.7:job-server
 :beam-runners-flink-1.7 :runners:flink:1.7
-:beam-runners-flink-1.6-job-server-container :runners:flink:1.6:job-server-container
-:beam-runners-flink-1.6-job-server :runners:flink:1.6:job-server
-:beam-runners-flink-1.6 :runners:flink:1.6
```
Comment (mxm) [inline project-mappings]: Looks good. Please also remove `:beam-runners-flink_2.11*` above.

Comment (sunjincheng121) [reply project-mappings]: Yes, we can remove the flink-1.5 as well in this PR.

Comment (sunjincheng121) [reply project-mappings]: I think the `` is mistake of the commit `https://github.com/apache/beam/pull/9632/files#diff-2bb618406f7ee4470a48343283f368a2R97`
So, I think we do not need remove it, and  it's better to correct the Artifact Id for flink-1.8.
What do you think?

**Comment by sunjincheng121** [comment]
Thanks for the review @mxm!
I have update the PR, I appreciate if you have another look.
Best,
Jincheng

**Comment by adude3141** [comment]
LGTM.

But we might consider to close this in favour of https://github.com/apache/beam/pull/8915

@lukecwik what do you think?

File: project-mappings (hunk 1 above)
Comment (mxm) [reply project-mappings]: Makes sense to me.

File: project-mappings [hunk 2]
```diff
@@ -94,15 +94,12 @@
 :beam-runners-google-cloud-dataflow-java-examples :runners:google-cloud-dataflow-java:examples
 :beam-runners-google-cloud-dataflow-java :runners:google-cloud-dataflow-java
 :beam-runners-gearpump :runners:gearpump
-:beam-runners-flink_2.11-job-server-container :runners:flink:1.8:job-server-container
-:beam-runners-flink_2.11-job-server :runners:flink:1.8:job-server
-:beam-runners-flink_2.11 :runners:flink:1.8
+:beam-runners-flink_1.8-job-server-container :runners:flink:1.8:job-server-container
+:beam-runners-flink_1.8-job-server :runners:flink:1.8:job-server
+:beam-runners-flink_1.8 :runners:flink:1.8
```
Comment (mxm) [inline project-mappings]:
```suggestion
:beam-runners-flink-1.8 :runners:flink:1.8
```

File: project-mappings [hunk 3]
```diff
@@ -94,15 +94,12 @@
 :beam-runners-google-cloud-dataflow-java-examples :runners:google-cloud-dataflow-java:examples
 :beam-runners-google-cloud-dataflow-java :runners:google-cloud-dataflow-java
 :beam-runners-gearpump :runners:gearpump
-:beam-runners-flink_2.11-job-server-container :runners:flink:1.8:job-server-container
-:beam-runners-flink_2.11-job-server :runners:flink:1.8:job-server
-:beam-runners-flink_2.11 :runners:flink:1.8
+:beam-runners-flink_1.8-job-server-container :runners:flink:1.8:job-server-container
+:beam-runners-flink_1.8-job-server :runners:flink:1.8:job-server
```
Comment (mxm) [inline project-mappings]:
```suggestion
:beam-runners-flink-1.8-job-server :runners:flink:1.8:job-server
```

File: project-mappings [hunk 4]
```diff
@@ -94,15 +94,12 @@
 :beam-runners-google-cloud-dataflow-java-examples :runners:google-cloud-dataflow-java:examples
 :beam-runners-google-cloud-dataflow-java :runners:google-cloud-dataflow-java
 :beam-runners-gearpump :runners:gearpump
-:beam-runners-flink_2.11-job-server-container :runners:flink:1.8:job-server-container
-:beam-runners-flink_2.11-job-server :runners:flink:1.8:job-server
-:beam-runners-flink_2.11 :runners:flink:1.8
+:beam-runners-flink_1.8-job-server-container :runners:flink:1.8:job-server-container
```
Comment (mxm) [inline project-mappings]:
```suggestion
:beam-runners-flink-1.8-job-server-container :runners:flink:1.8:job-server-container
```

**Comment by sunjincheng121** [comment]
Thanks @mxm @adude3141.

**Comment by mxm** [comment]
Closing this since #8915 has been merged. Thanks for opening the PR @sunjincheng121. I think it led to merging the aforementioned PR ;)

**Pull request closed without merging**